### DISCUSSION TITLE
Issue of shortcuts of Quarto

### DISCUSSION THREAD

**Discussion by reddylee** [quarto-dev/quarto-cli#6336] [Q&A]
### Description

Hi, guys. I got a strange problem. Some shortcuts work in Rmarkdown, but not in Quarto. For example, Insert Assignment Operator, Insert Pipe Operator. Anyone runs into the same problem?

**Comment by mcanouil** [comment]
Could you remove the `<>`from the title and give more details, as currently there is no way for us to even understand your problem or your setup. What editor? What operating system? etc.

**Reply by reddylee** [reply]
Hi, sorry for the trouble. I fixed the issue by updating quarto to the latest version.

**Comment by reddylee** [comment]
Update to the latest version of Quarto.
